### PR TITLE
RM-24092 update current time indicator to fallback one hour on daylightsavings…

### PR DESCRIPTION
**Pull request by kbbushman**
**Changes included in this pull request:**

- Update current time indicator to check for DST fallback

**Fixes JIRA issues:**
- Ticket: [RM-24092](https://elationhealth.atlassian.net/browse/RM-24092)

Always follow the [contribution guidelines](https://github.com/jquense/react-big-calendar/blob/master/CONTRIBUTING.md) when submitting a pull request.

**Before FIx:**
Screen Shot 2022-11-06 at 11 55 50 AM

**After Fix:**
Screen Shot 2022-11-06 at 11 56 45 AM

[RM-24092]: https://elationhealth.atlassian.net/browse/RM-24092?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/TimeColumn.js [hunk 1]
```diff
@@ -129,7 +130,14 @@ export default class TimeColumn extends Component {
...
 
-    const now = dates.now(nowTimezone);
+    let now = dates.now(nowTimezone);
+
+    const daylightSavingsShift = getDaylightSavingsShift(now)
+    const isFallingBack = isDaylightSavingsFall(daylightSavingsShift);
+
+    if (isFallingBack) {
+      now = dates.add(now, daylightSavingsShift, 'minutes');
+    }
```
Comment (sidneytang-elation) [inline src/TimeColumn.js:140]: I'm not sure this is the right condition - don't we need to check whether the current time is pre-time change or post-time change (i.e. 2AM)? This code will always shift the indicator position as long as it's on a Sunday of fall daylight savings, but my understanding is that the indicator is only positioned incorrectly on one side of that time change.

Comment (kbbushman) [reply src/TimeColumn.js:140]: Thanks! Yeah, it should only be off after 2:00AM. I'll update the condition.

File: src/TimeColumn.js [hunk 2]
```diff
@@ -129,7 +130,25 @@ export default class TimeColumn extends Component {
...
+
+    const dayStart = dates.startOf(now, 'day');
+    const dayEnd = dates.endOf(now, 'day');
+
+    const dayStartTimezoneOffset = dayStart.getTimezoneOffset();
+    const dayEndTimezoneOffset = dayEnd.getTimezoneOffset();
+    const nowTimezoneOffset = now.getTimezoneOffset();
+
+    // This is generally at 2:00am on the day we fall back to standard time
+    const isAfterFallBackTimeShift =
```
Comment (sidneytang-elation) [inline src/TimeColumn.js:143]: Isn't the first part of this condition the same thing as `isFallingBack`? Can we do this instead?
```ts
const isAfterFallBackTimeShift = isFallingBack && nowTimezoneOffset === dayEndTimezoneOffset;
```
I suppose I'm not sure we even need to care about `dayStartTimezoneOffset`.

Comment (sidneytang-elation) [reply src/TimeColumn.js:143]: Then the condition on line 149 would just be `if (isAfterFallBackTimeShift)`

Comment (kbbushman) [reply src/TimeColumn.js:143]: Yeah, the current solution is more specific than it needs to be. I'll clean it up.

**Review by sidneytang-elation — DISMISSED** [review]

**Review by sidneytang-elation — APPROVED** [review]

**Pull request merged**